### PR TITLE
Add LQ images for progressive image loading

### PR DESCRIPTION
**Pull request by LudvigHz**
~This field can be used to automatically create a placeholder image beside the full image, that can be used as a placeholder.~

Adds plenty of `*_placeholder` images used for progressive image loading.

### CODE REVIEW

**Comment by LudvigHz** [comment]
### UPDATE:
Now with less content shift and more/better loading. I have gone through and most places now should not shift content at all when partially loading.

Also: there were some really wonky stuff with the articles page that I hadn't noticed before, like misaligned items and weird borders. Should be good now :smile: 

https://user-images.githubusercontent.com/42850232/153245458-51543555-5fa4-4401-ad07-b7b5fa578336.mp4

https://user-images.githubusercontent.com/42850232/153245591-9514ea45-e486-456f-ab67-6354ca161527.mp4

File: lego/apps/articles/serializers.py [hunk 1]
```diff
@@ -75,6 +79,9 @@ class Meta:
 class PublicArticleSerializer(TagSerializerMixin, BasisModelSerializer):
 
     cover = ImageField(required=False, options={"height": 300})
+    cover_placeholder = ImageField(
+        source="cover", required=False, options={"height": 50, "filters": ["blur(20)"]}
```
Comment (SmithPeder) [inline lego/apps/articles/serializers.py:83]: this should be 30 ey?

**Review by SmithPeder — CHANGES_REQUESTED** [review]
If i get the schema correctly, my comments make sense?

File: lego/apps/companies/serializers.py [hunk 2]
```diff
@@ -86,6 +86,9 @@ def create(self, validated_data):
 
 class CompanyListSerializer(BasisModelSerializer):
     logo = ImageField(required=False, options={"height": 500})
+    logo_placeholder = ImageField(
+        source="logo", required=False, options={"height": 500, "filters": ["blur(20)"]}
```
[the remark this answers is not in the document]
Comment (LudvigHz) [reply lego/apps/companies/serializers.py:90]: oh damn. yup

File: lego/apps/articles/serializers.py (hunk 1 above)
Comment (LudvigHz) [reply lego/apps/articles/serializers.py:83]: yup

**Comment by LudvigHz** [comment]
> Shouldn't the SearchArticleSerializer class also use a cover placeholder?

No. This is intentional, as this serializer is only used when searching, where we don't use the cover. If you think it should, sure. But I don't think it needs to be there. (We don't use the cover on lego-webapp at least).

**Review by SmithPeder — APPROVED** [review]

**Pull request merged**